perk-store-services/src/main/java/org/exoplatform/perkstore/dao/PerkStoreOrderDAO.java

```diff
@@ -167,8 +167,8 @@ public double countUserOrderedQuantityByStatusInPeriod(long productId,
     return result == null ? 0 : result;
   }
 
-  public List<ProductOrderEntity> getOrders(String username, OrderFilter filter) {
-    StringBuilder orderQuery = getOrderFilterQueryString(username,filter);
+  public List<ProductOrderEntity> getOrders(String username, OrderFilter filter, Boolean isPerkStoreManager, Boolean isProductOwner) {
```

Comment:
```suggestion
  public List<ProductOrderEntity> getOrders(String username, OrderFilter filter, boolean isPerkStoreManager, boolean isProductOwner) {
```
If we don't use null value, it's better to use primitive value in method signature to avoid conversions from & to Object.